test/webgl/uniforms.spec.js

```diff
@@ -64,7 +64,10 @@ uniform sampler2D s2d;
 
 void main(void) {
   vec4 v = vec4(f) + vec4(v2, 0., 0.) + vec4(v3, 0.) + v4;
-  ivec4 iv = ivec4(i) + ivec4(iv2, 0., 0.) + ivec4(iv3, 0.) + iv4;
+  ivec4 iv = ivec4(i, 0, 0, 0);
```

Comment:
Nit: Add a comment explaining why we are coding it like this. If not, the next maintainer might just restore it to previous state...